creusot/src/backend/logic.rs

```diff
@@ -22,6 +23,9 @@ pub(crate) fn translate_logic(ctx: &Why3Generator, def_id: DefId) -> Option<File
     let mut names = Dependencies::new(ctx, def_id);
     let pre_sig = ctx.sig(def_id).clone().normalize(ctx.tcx, ctx.typing_env(def_id));
 
+    let namespace_ty =
+        names.def_ty_no_dependency(get_namespace_ty(ctx.ctx.tcx), GenericArgsRef::default());
```

Comment:
Why don't you want the dependency? This seems a bit fishy.

Reply:
I see. This is kind of a dirty hack, but it is difficult to do better.

Perhaps this should be done as part of the code of the expander.... Not sure what's better.